src/services/Store/Contact/useContacts.tsx

```diff
@@ -118,7 +134,9 @@ function useContacts() {
     contacts,
     contactRestore,
     createContact,
+    createContacts,
```

Comment:
We are working to remove the hooks and let the components connect directly to the store.